store_monitoring/crud.py

```diff
@@ -16,14 +18,22 @@ def insert_store_status(db):
             db.refresh(data)
     return
 
+def string_to_time(time):
+    if(len(time)>7):
+        time=datetime.strptime(time.replace(".",":"), "%H:%M:%S")
+    else:
+        time="0"+time
+        time = datetime.strptime(time.replace(".",":"), "%H:%M:%S")
+    return (time.time())
+
 def insert_menu_hours(db):
     with open (MENU_HOURS_FILE_PATH, mode='r', encoding='utf-8-sig') as file:
         file_content=csv.DictReader(file)
         for row in file_content:
             store_id=row["store_id"]
             day=row["day"]
-            start_time_local=row["start_time_local"]
-            end_time_local=row["end_time_local"]
+            start_time_local=string_to_time(row["start_time_local"])
```

Comment:
You haven't thought about converting it to utc. I'd say keep all the times in utc, so that while querying it's easier for you.

At that time you won't have to think about changing time zone etc